docs/en/administration/Upgrade.md

```diff
@@ -87,6 +87,150 @@ $ ./bin/alluxio-start.sh all
 ```
 5. If you have updated the Alluxio client jar for an application, restart that application to use the new Alluxio client jar.
 
+### Rolling upgrade/restart masters
```

Comment:
the rolling upgrade feature should be considered as experimental. i'll make a new grouping and tag it